src/components/Masthead/SessionLinks.js

```diff
@@ -28,6 +28,7 @@ class SessionLinks extends React.Component {
           return (
             <Link
               id={`session-links-${i}`}
+              key={i}
```

Comment:
It's usually a good idea to add some sort of prefix to the index when using it as a React key. That way we know it's unique across all of the rendered JSX. Otherwise React's diffing algorithm will bail from updating that individual item and just re-render everything in that loop.

key={`some-unique-string-${i}`}